trin-core/src/portalnet/overlay.rs

```diff
@@ -194,6 +195,10 @@ impl OverlayProtocol {
         distances: Vec<u16>,
     ) -> Result<Nodes, OverlayRequestError> {
         // Construct the request.
+        match validate_find_nodes_distances(&distances) {
+            Ok(_) => (),
+            Err(msg) => return Err(msg),
+        }
```

Comment:
```suggestion
        validate_find_nodes_distances(&distances)?;
```
I guess it works here, too?